packages/direflow-component/src/WebComponentFactory.tsx

```diff
@@ -60,6 +63,7 @@ class WebComponentFactory {
 
     return class extends HTMLElement {
       private _application: JSX.Element | undefined;
+      private _properties: any = clonedeep(factory.componentProperties);
```

Comment:
dont very much like the bland any time here. 
would preffer `{ [key: string]: any }` since object?

Reply:
Sure, we can go with `{ [key: string]: any }`